pmlc/conversion/tile_to_pxa/tests/reshape.mlir

```diff
@@ -0,0 +1,59 @@
+// RUN: pmlc-opt -convert-tile-to-pxa -canonicalize -cse %s | FileCheck %s
+
+module {
+  func @reshaper(%arg0: tensor<1x1x60xf32>) -> tensor<60xf32> {
+    %c60 = tile.constant 60
+    %0 = "tile.reshape"(%arg0, %c60) : (tensor<1x1x60xf32>, index) -> tensor<60xf32>
+    return %0 : tensor<60xf32>
+  }
+}
+
+// CHECK-LABEL: func @reshaper
+// CHECK: stdx.reshape
+// CHECK: memref<60xf32>
+
+// -----
+
+module {
+  func @reshaper(%arg0: tensor<2x4x5xf32>) -> tensor<2x20xf32> {
+    %c2 = tile.constant 2
+    %c20 = tile.constant 20
+    %0 = "tile.reshape"(%arg0, %c2, %c20) : (tensor<2x4x5xf32>, index, index) -> tensor<2x20xf32>
+    return %0 : tensor<2x20xf32>
+  }
+}
+
+// CHECK-LABEL: func @reshaper
+// CHECK: stdx.reshape
+// CHECK: memref<2x20xf32>
+
+// -----
+
+module {
+  func @reshaper(%arg0: tensor<5x6xf32>, %arg1: tensor<5x2x3xf32>) -> tensor<5x6xf32> {
+    %c5 = tile.constant 5
+    %c6 = tile.constant 6
+    %0 = "tile.reshape"(%arg1, %c5, %c6) : (tensor<5x2x3xf32>, index, index) -> tensor<5x6xf32>
+    return %0 : tensor<5x6xf32>
+  }
+}
+
+// CHECK-LABEL: func @reshaper
+// CHECK: stdx.reshape
+// CHECK: memref<5x6xf32>
```

Comment:
Could we add more to the test so that it verifies that the strides are being manipulated? Or am I misunderstanding what the `BaseViewConversionHelper` is doing?

Reply:
Oh, perhaps what I'm missing is that this is a tile -> stdx test. How about we have a stdx -> LLVM test as well?